versions/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+set -u
+
+# Make sure BUILD_DIR is set and default to website
+if [ -z ${BUILD_DIR+x} ]; then
+  echo "BUILD_DIR is not set, falling back to default"
+  export BUILD_DIR="website"
+else
+  echo "BUILD_DIR is set to '$BUILD_DIR'"
+fi
+
+# Get the package.json version
+PACKAGE_VERSION="$(grep \"version\" ./package.json | awk '{gsub(/[\",]/, ""); print $2}')"
+
+# Change into the website directory
+cd $BUILD_DIR;
+
+# Get the latest version of the docs
+DOCS_VERSION="$(cat versions.json | head -2 | tail -1 | awk '{gsub(/[",]/, ""); print $0}')"
+
+# Update the version of the docs if is necessary
+if ! [ "${DOCS_VERSION}" == "" ] && ! [ ${PACKAGE_VERSION} == ${DOCS_VERSION} ]; then
```

Comment:
Could we echo something if this fails? Maybe to set a log why it didn't happen

Reply:
Sure; we can add it